src/components/Footer.jsx

```diff
@@ -0,0 +1,35 @@
+import "./Footer.css";
+import youtube from "../assets/youtube.svg";
+import instagram from "../assets/insta.svg";
+import twitter from "../assets/twitter.svg";
+import facebook from "../assets/facebook.svg";
+import { FACEBOOK_URL, INSTAGRAM_URL, TWITTER_URL, YOUTUBE_URL } from "../api";
+
+const Footer = () => {
+  return (
+    <div className='Footer'>
```

Comment:
className 규칙 통일해주세요!
```suggestion
const Footer = () => {
  return (
    <div className='footer'>
```